### ISSUE TITLE
Wait method (Task) to finish?

### ISSUE THREAD

**Issue by mharj** [bhoriuchi/vsphere-connect#11]
Seems that on old API there was client.monitor.task to track Task to finish (i.e. CreateSnapshot_Task).
Is there some similar way to track Task status with new API?

**Comment by bhoriuchi** [comment]
yeah, for currently supported operations that return tasks you should be able to pass the `{ async: false }` option

see https://github.com/bhoriuchi/vsphere-connect/blob/master/src/methods/destroy.js#L12-L14
